App/UI/CountriesOfInterest/CountriesOfInterestVC.swift

```diff
@@ -0,0 +1,105 @@
+// CountriesOfInterestVC.swift
+// Copyright (C) 2020 Presidenza del Consiglio dei Ministri.
+// Please refer to the AUTHORS file for more information.
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+import Foundation
+import Models
+import Tempura
+
+final class CountriesOfInterestVC: ViewControllerWithLocalState<CountriesOfInterestView> {
+  override func setupInteraction() {
+    self.rootView.userDidTapClose = { [weak self] in
+      self?.store.dispatch(Hide(Screen.updateCountry, animated: true))
+    }
+
+    self.rootView.userDidTapComplete = { [weak self] in
+      self!.store.dispatch(Logic.Settings.CompleteUpdateCountries(newCountries: self!.localState.currentCountries))
```

Comment:
<!--
  0 failure: 
  2 warning:  swiftlint (0.39.2..., swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: App/UI/CountriesOfInterest/CountriesOfInterestVC.swift;
  Line: 26;
-->

- :warning: swiftlint (0.39.2): Force unwrapping should be avoided. (force_unwrapping)
- :warning: swiftlint (0.39.2): Force unwrapping should be avoided. (force_unwrapping)